Euler.ipynb

```diff
@@ -342,7 +346,7 @@
     "\n",
     "1. Define a piecewise function giving the middle state velocity $u_m$ that can be connected to the left state by an entropy-satisfying shock or rarefaction, as a function of the middle-state pressure $p_m$.\n",
     "2. Define a piecewise function giving the middle state velocity $u_m$ that can be connected to the right state by an entropy-satisfying shock or rarefaction, as a function of the middle-state pressure $p_m$.\n",
-    "3. Use an iterative solver to find the intersection of the two functions defined above.\n",
+    "3. Use an nonlinear rootfinder to find the intersection of the two functions defined above.\n",
```

Comment:
I guess this should be "a nonlinear rootfinder".

Reply:
Yes, thanks.  I changed this wording for consistency with Shallow_water.